core/common/src/main/java/com/alipay/sofa/rpc/common/config/RpcConfigKeys.java

```diff
@@ -102,4 +102,13 @@ public class RpcConfigKeys {
                                                                                         false,
                                                                                         "judge the generic object exception fields.",
                                                                                         new String[] { "sofa_rpc_generic_exception_fields" });
+
+    // config for unique-id check
+    public static final ConfigKey<Boolean>  UNIQUE_ID_VALIDATE                  = ConfigKey
+                                                                                    .build(
+                                                                                        "sofa.rpc.unique-id.check",
```

Comment:
This naming suggestion is `sofa.rpc.uniqueId.pattern.check` .

Reply:
get